frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -6,16 +6,19 @@ import { Account } from "../../libraries/explorer-wamp/accounts";
 
 import Balance from "../utils/Balance";
 import CardCell from "../utils/CardCell";
+import BlockLink from "../utils/BlockLink";
+import TransactionLink from "../utils/TransactionLink";
 
 export interface Props {
   account: Account;
+  blockHash: string;
 }
 
-export default ({ account }: Props) => {
+export default ({ account, blockHash }: Props) => {
```

Comment:
Let's stop for a moment and think about the context.

Imagine you are opening this file the very first time in your life and you know that it is an AccountDetails component. Do you expect account infromation being passed via props? Probably, yes. Do you expect some random `blockHash` being passed? I would certainly cast "WTF?!".

I want to ask you to think about this puzzle yourself.

Reply:
change the account type include the hash